routers/web/repo/fork_to_edit.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2016 The Gogs Authors. All rights reserved.
```

Comment:
If this copied from Gogs?

```suggestion
// Copyright 2025 The Gitea Authors. All rights reserved.
```

Reply:
There is a bit of code moved from the existing `editor.go`, so I added an additional copyright line instead of replacing it.